src/main/main.ts

```diff
@@ -148,6 +159,49 @@ ipcMain.on(ChannelEnum.HEALTH_CHECK, async (event) => {
   }
 });
 
+ipcMain.on(ChannelEnum.LOAD_PRESET, async (event, preset_name: string) => {
+  const channel = ChannelEnum.LOAD_PRESET;
+  console.log(`Loading preset: ${preset_name}`);
+
+  // TODO: should we do some str checking here?
+  // might have to make the preset folder too?
```

Comment:
We can clean up these comments too. We don't seem to be doing str checking per se but we ARE throwing an error if we fail to fetch so that seems good enough to me.